jest.config.js

```diff
@@ -0,0 +1,18 @@
+const packageJSON = require('./package.json');
+
+process.env.TZ = 'UTC';
+
```

Comment:
this is useful to avoid dealing with dates changing when running under CI systems.